### ISSUE TITLE
Vertical page scroll not available both in view and in edit mode

### ISSUE THREAD

**Issue by ananddass** [Ink/ThatPDF#4]
Users expect vertical scroll of the document but currently only horizontal swipe is available in both the view and edit modes

**Comment by brettcvz** [comment]
Will be solved via https://github.com/Ink/ThatPDF/issues/7

**Comment by steipete** [comment]
Right. PSPDFKit supports horizontal/vertical modes (http://pspdfkit.com/documentation/Classes/PSPDFViewController.html#//api/name/scrollDirection) and paged/continuous and page curl transition. (http://pspdfkit.com/documentation/Classes/PSPDFViewController.html#//api/name/pageTransition)
